ruma-push-gateway-api/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# [Unreleased]
+
+Improvements:
+
+* Add endpoints:
+
+  `push::send_event_notification::v1`
```

Comment:
Since the crate only consists of a single endpoint, I think the multiline format isn't really needed here. I guess we can keep it for some slight consistency? If you like the current version feel free to just close this review thread.